src/index.html

```diff
@@ -7,6 +7,303 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </div>
+      <nav class="menu">
+        <ul class="menu_list">
+          <li class="menu_item">
+            <a
+              class="menu_link is-active"
+              href="#">
+                Apple
+            </a>
+          </li>
+          <li class="menu_item">
+            <a class="menu_link" href="#">
+              Samsung
+            </a>
+          </li>
+          <li class="menu_item">
+            <a class="menu_link" href="#">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="menu_item">
+            <a
+              class="menu_link"
+              data-qa="nav-hover"
+              href="#">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="menu_item">
+            <a class="menu_link" href="#">
+              Гаджеты
+            </a>
+          </li>
+          <li class="menu_item">
+            <a class="menu_link" href="#">
+            Планшеты
+            </a>
+          </li>
+          <li class="menu_item">
+            <a class="menu_link" href="#">
+              Фото
+            </a>
+          </li>
+          <li class="menu_item">
+            <a class="menu_link link_video" href="#">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <section class="cards">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          >
+          <p class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__feedback">
+            <div class="card__stars">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+            </div>
+            <p class="card__reviews">Отзывов: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price_text">Цена:</p>
+            <p class="card__price_number">69 999 грн</p>
```

Comment:
одно нижнее подчеркивание это модификатор '_'
```suggestion
            <p class="card__price-text">Цена:</p>
            <p class="card__price-number">69 999 грн</p>
```